docs/source/users/index.md

```diff
@@ -7,27 +7,102 @@ please see our {doc}`contributor's guide </contributors/index>`.
 ```
 
-To use the `jupyter_ai` package, you will also need to have a currently-maintained version of JupyterLab 3 installed. We do not yet support JupyterLab 4. If you use `conda`, you can install JupyterLab in your environment by running:
+The `jupyter_ai` package, which provides the lab extension and user interface in
+JupyterLab, depends on JupyterLab 4. If upgrading to JupyterLab 4 is not
+possible in your environment, you can instead install `jupyter_ai` v1.x instead.
+See "Installation" for more details.
+
+JupyterLab may be installed two ways.
+
+1. via `pip`:
+
+```
+# change 4.0 to 3.0 if you need JupyterLab 3
+pip install jupyterlab~=4.0
+```
+
+2. via `conda`:
 
 ```
-conda install jupyterlab
+# change 4.0 to 3.0 if you need JupyterLab 3
+conda config --add channels conda-forge
+conda config --set channel_priority strict
+conda install jupyterlab~=4.0
 ```
 
-You should have Jupyter Server 2.x (not 1.x) installed. A fresh install of JupyterLab 3.6.x should come with Server 2.x. You can find your current Server version by running `jupyter --version` and checking for a line beginning with `jupyter_server`. To upgrade your version of Jupyter Server, run:
+The `jupyter_ai_magics` package, which provides exclusively the IPython magics,
+does not depend on JupyterLab or `jupyter_ai`. `jupyter_ai_magics` may be
+installed without `jupyterlab` or `jupyter_ai`.
+
+## Installation
+
+### Installation via `pip`
+
+If you want to install the JupyterLab extension, you can run:
 
 ```
-pip install jupyter_server --upgrade
+pip install jupyter_ai
 ```
 
-You can use the `jupyter_ai_magics` package without JupyterLab, but you will need a compatible interface, such as [IPython](https://ipython.org/).
+The latest major version of `jupyter_ai` is currently v2, and supports
+exclusively JupyterLab 4.  If you need support for JupyterLab 3, you should use
+`jupyter_ai` v1 instead, which can be installed via:
+
+```
+pip install jupyter_ai~=1.0
+```
+
+If you are not using JupyterLab and you only want to install the Jupyter AI `%%ai` magic, you can run:
+
+```
+$ pip install jupyter_ai_magics
+```
+
+`jupyter_ai` depends on `jupyter_ai_magics`, so installing `jupyter_ai`
+automatically installs `jupyter_ai_magics`.
+
+### Installation via `pip` within Conda environment (recommended)
+
+We highly recommend installing both JupyterLab and Jupyter AI within an isolated
+Conda environment to avoid clobbering Python packages in your existing Python
+environment.
+
+First, install
+[conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html)
+and create an environment that uses Python 3.11:
+
+    $ conda create -n jupyter-ai python=3.11
+    $ conda activate jupyter-ai
+    $ pip install jupyter_ai
+
+Then, the steps from "Requirements" and "Installation via `pip`" may be followed
+to install JupyterLab and Jupyter AI within this Conda environment.
+
+When starting JupyterLab with Jupyter AI, make sure to activate the Conda
+environment first:
```

Comment:
I'd mention the need to make sure appropriate conda environment is active in the same spot conda is first mentioned on the page, in Prerequisites (or conda installation - Prerequisites if structure suggested in my comment above would be used)